crates/oxc_parser/src/modifiers.rs

```diff
@@ -178,20 +252,52 @@ impl ModifierKind {
         }
     }
 }
+impl From<Kind> for ModifierKind {
+    // NOTE (@DonIsaac) should we be using TryFrom here? or does unreachable!()
```

Comment:
`unreachable` is too unsafe here, better to return a None, and return unexpected token on the call site.

Need to be extra safe for a parser.

Reply:
Fixed